Datadog.Trace.sln

```diff
@@ -1515,7 +1517,7 @@ Global
 		{2CC63AEB-0098-4D3B-9606-F07692C03E90} = {BAF8F246-3645-42AD-B1D0-0F7EAFBAB34A}
 		{DEACDE01-95FE-4777-B70A-F20A96AABEA7} = {BAF8F246-3645-42AD-B1D0-0F7EAFBAB34A}
 		{754F73E1-F7A4-47C7-A3F7-DC59ADA5105A} = {BAF8F246-3645-42AD-B1D0-0F7EAFBAB34A}
-		{F6A03B6C-EBF9-4581-9904-EDC7270CF3BD} = {0972AD57-B16B-494F-AE0A-091DD6F3B42B}
+		{F6A03B6C-EBF9-4581-9904-EDC7270CF3BD} = {72A77450-6A42-4302-8BDE-0D5878EC7BCA}
```

Comment:
Are these changes necessary?

Reply:
Not necessary, but reorder some samples in the solution